frontend/public/components/_nav-title.scss

```diff
@@ -28,3 +28,13 @@
     padding-top: ($grid-gutter-width / 2);
   }
 }
+
+.co-m-nav-title--overview {
```

Comment:
Nit:  nest this in the above rule so it's consistent with the other modifier rules.